lib/rfc5424.js

```diff
@@ -5,30 +5,9 @@ const stringify = require('fast-safe-stringify')
 const through2 = require('through2')
 const utils = require('./utils')
 
-let options
-
-function buildMessage (data) {
-  let message = {}
-  if (options.messageOnly) {
-    message = data.msg
-    return message
-  }
-
-  if (options.includeProperties.length > 0) {
-    options.includeProperties.forEach((p) => { message[p] = data[p] })
-    message = stringify(message)
-  } else {
-    message = stringify(data)
-  }
-
-  return message
-}
-
-module.exports = function ($options) {
-  options = $options
+function messageBuilderFactory (options) {
```

Comment:
Same as other spec file. Please explain.

Reply:
This file has the same changes as the previous one:

- removed global option object that was shared across functions
- isolated the function to build the message string to write out
- expose the build message function